ethcore/res/ethereum/kovan.json

```diff
@@ -49,7 +49,8 @@
 		"eip1014Transition": "0x8c6180",
 		"eip1052Transition": "0x8c6180",
 		"eip1283Transition": "0x8c6180",
-		"kip4Transition": "0x8c6180",
+		"eip1283DisableTransition": "0x8c6180",
```

Comment:
I suppsoe we want to keep `kip4` at the same block and just schedule disabling of `eip1283`? The numbers should be swapped.

Reply:
Good someone is actually looking at the specs :+1: